5 view individual article

edeharrison opened the pull request.

Commit:
setup SingleArticle component

Commit:
added SingleArticle component

Commit:
built out /articles/:article_id, moved Topics component inside Articles component

Slyose reviewed (approved):
Looks great Ede! Happy to see this PR merged - after lowering the states and useEffect in your app.js to be component specific

Commit:
moved articles state down into Articles component

edeharrison commented:
> Looks great Ede! Happy to see this PR merged - after lowering the states and useEffect in your app.js to be component specific

I moved articles state down from App into Articles
I moved the getArticles useEffect down from App into Articles

I left singleArticle state up in App because Articles (child to App) needs access to it
